scripts/cilium_migration.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
```

Comment:
@Pankhuri-Rajesh I'm trying this out on a system and it looks like this script currently has 644 permission, could you make it executable?